### PR TITLE
Use SAFEHOSTPLATFORM for olm-bundle URL

### PR DESCRIPTION
**Pull request by muvaf**

### CODE REVIEW

File: makelib/common.mk [hunk 1]
```diff
@@ -132,7 +132,6 @@ endif
 # Standardize Host Platform variables
 HOST_PLATFORM := $(HOSTOS)_$(HOSTARCH)
 SAFEHOSTPLATFORM := $(HOSTOS)-$(SAFEHOSTARCH)
```
Comment (muvaf) [inline makelib/common.mk:134]: @AaronME is there a specific reason both `SAFEHOST_PLATFORM` and `SAFEHOSTPLATFORM` exists? And would you have a preference between them?

File: makelib/k8s_tools.mk [hunk 2]
```diff
@@ -97,34 +97,34 @@ $(KUBECTL):
 $(KUSTOMIZE):
 	@$(INFO) installing kustomize $(KUSTOMIZE_VERSION)
 	@mkdir -p $(TOOLS_HOST_DIR)/tmp-kustomize
-	@curl -fsSL https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/$(KUSTOMIZE_VERSION)/kustomize_$(KUSTOMIZE_VERSION)_$(SAFEHOST_PLATFORM).tar.gz | tar -xz -C $(TOOLS_HOST_DIR)/tmp-kustomize
+	@curl -fsSL https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/$(KUSTOMIZE_VERSION)/kustomize_$(KUSTOMIZE_VERSION)_$(SAFEHOSTPLATFORM).tar.gz | tar -xz -C $(TOOLS_HOST_DIR)/tmp-kustomize
```
Comment (hasheddan) [inline makelib/k8s_tools.mk:100]: Will this break since these two variables are not interchangeable?

File: makelib/common.mk (hunk 1 above)
Comment (hasheddan) [reply makelib/common.mk:134]: @muvaf note that one use `-` and one uses `_`

File: makelib/k8s_tools.mk (hunk 2 above)
Comment (AaronME) [reply makelib/k8s_tools.mk:100]: Based on https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh the kustomize install script is setting the version to "${opsys}_${arch}" -- so SAFEHOST_PLATFORM should be correct.

File: makelib/common.mk (hunk 1 above)
Comment (muvaf) [reply makelib/common.mk]: Wow I did not notice that, reverted the removal. Thanks folks!

**Review by turkenh — APPROVED** [review]

**Pull request merged**